projects/perception/facial_expression_recognition/image_based_facial_emotion_estimation/README.md

```diff
@@ -0,0 +1,96 @@
+# Skeleton-based Human Action Recognition Demo
```

Comment:
Please check the title again. Also, should this called image_based_facial_expression_recognition or ensemble_based_facial_expression_recognition? Both are used, which can be a bit confusing